repo/content/content_manager.go

```diff
@@ -924,9 +929,9 @@ func (bm *WriteManager) lock() {
 }
 
 // +checklocksrelease:bm.mu
-func (bm *WriteManager) unlock() {
+func (bm *WriteManager) unlock(ctx context.Context) {
```

Comment:
thinking out loud: this seems like an _artificial artifact_, not clear anything simple can be done about it.